internal/db/tx.go

```diff
@@ -115,10 +116,42 @@ func (tx *holdTx) Connect(source, target, rel ID) error {
 	return nil
 }
 
+func (tx *holdTx) Disconnect(source, target, rel ID) error {
+	tx.ensureWrite()
+	tx.h = tx.h.Unlink(source, target, rel)
+	return nil
+}
+
 func (tx *holdTx) Delete(rec Record) error {
 	tx.ensureWrite()
+	rels, err := rec.Interface().Relationships()
+	if err != nil {
+		return err
+	}
+	for _, rel := range rels {
+		if rel.Multi() {
```

Comment:
should this just be part of the hold?

Reply:
I'm not exactly clear on what wants to be at the hold level vs. the tx level. Fine to put it in the hold

Reply:
Also, it doesn't look like the hold has a notion of multi, so i'm gonna keep this here for now!